src/pages/CoursePageViewWithVideo/IndividualCourse.js

```diff
@@ -88,8 +89,16 @@ export default function CoursePageViewWithVideo(props) {
       title: "About Course",
       onPress: () => console.log("About Course"),
     },
+
     {
       id: 6,
```

Comment:
@sakshi-choudhary I reviewed your code and I think the `build CI` failed because of this extra space as it may violated the prettier(code Formatter) code style rule.

Reply:
thanks